ddtrace/internal/telemetry/writer.py

```diff
@@ -204,6 +209,28 @@ def app_started_event(self):
         }
         self.add_event(payload, "app-started")
 
+    def app_heartbeat_event(self):
+        # type: () -> None
+        if self._forked:
+            # TODO: Enable app-heartbeat on forks
+            #   Since we only send app-started events in the main process
+            #   any forked processes won't be able to access the list of
+            #   dependencies for this app, and therefore app-heartbeat won't
+            #   add much value today.
+            return
+
+        # DEV: Although the default flush interval is 60 seconds,
+        #   we want to explicitly control the flow here in case
+        #   the flush interval ever changes.
+        #
+        #   If the flush interval ever exceeds 60 seconds, then we
```

Comment:
but if it's ever less, then we're only gonna have heartbeats at the flush interval?

Reply:
if the flush interval is < 60 seconds, we'll only create heartbeats every 60 seconds (~ish)

if the flush interval is ever >= 60 seconds, then we'll create one heartbeat per flush

Reply:
maybe worth renaming that variable to `HEARTBEAT_INTERVAL_MIN_INTERVAL` then?

Reply:
The minimum heartbeat interval is 60 seconds. The true value is highly dependant on the telemetry writer interval.

For example if `DD_INSTRUMENTATION_TELEMETRY_INTERVAL_SECONDS=30` we will send a heartbeat every other flush (so every 60 seconds). But if `DD_INSTRUMENTATION_TELEMETRY_INTERVAL_SECONDS=40` (its not divisible by 60) then the heartbeat interval will be 80 seconds.